packages/template-retail-react-app/app/components/product-tile/index.jsx

```diff
@@ -156,6 +157,7 @@ const ProductTile = (props) => {
                     />
                 </Box>
             )}
+            <BadgeGroup product={product?.representedProduct} />
```

Comment:
The `representedProduct` is an interesting [type](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-search?meta=type%3AProductSearchHit), because it's not really a product, but we are loosely saying it is a product type as we call the prop "product" .. I'd make sure that in the components JSDoc def we talk about this property, is we plan on passing in a complex object (which I don't think we should do haha)